docs/workflows.rst

```diff
@@ -64,3 +67,59 @@ All the commands should be run on Pulp 3 machine.
             }
         ]
     }
+
+
+Reset migrated Pulp 3 data
+--------------------------
+
+.. note::
+    Nothing is getting reset or modified in any way in Pulp 2.
+
+.. note::
+    Migrated artifacts are untouched. If you are sure, you want to remove them, please run the
```

Comment:
s/If you are sure, you want to remove them/If you are sure you want to remove them/